contrib/lib/Cargo.toml

```diff
@@ -16,6 +16,12 @@ json = ["serde", "serde_json"]
 msgpack = ["serde", "rmp-serde"]
 tera_templates = ["tera", "templates"]
 handlebars_templates = ["handlebars", "templates"]
+databases = ["r2d2", "diesel", "rocket_contrib_codegen", "rocket_contrib_codegen/database_attribute"]
```

Comment:
`diesel` is now always a requirement for database support, even when using non-diesel connections/pools

Reply:
Yes, yes it is. Probably not the intended behavior. This can be changed.